modules/claims_api/lib/bd/bd.rb

```diff
@@ -105,7 +105,7 @@ def generate_upload_body(claim:, doc_type:, pdf_path:, action:, original_filenam
       payload = {}
       auth_headers = claim.auth_headers
       veteran_name = compact_veteran_name(auth_headers['va_eauth_firstName'], auth_headers['va_eauth_lastName'])
-      birls_file_num = determine_birls_file_number(doc_type, auth_headers)
+      birls_file_num = determine_birls_file_number(doc_type, auth_headers) if %w[L075 L190].include?(doc_type)
```

Comment:
we would always get `nil` here with these conflicting lines, right? Between :108 and [:127](https://github.com/department-of-veterans-affairs/vets-api/blob/0a759a030d310cb1613389b3671cf3fa731ce0b6/modules/claims_api/lib/bd/bd.rb#L127)